docs/recipes/multiful-form-repo-fast-switching.md

```diff
@@ -0,0 +1,18 @@
+# multiful form repo Fast Switching n& maintain
```

Comment:
I am not sure I understand what you mean by this sentence? 😅

Reply:
Sorry, My Fault

Reply:
what do you mean by "multiful form repo"?

Reply:
Just taking a guess:
Is it "multiple Foam repos"?

Reply:
@ingalless yeah, sorry. @riccardoferretti Ingalless's description is right. English is my second language and almost not using in daily. only using internet things. (yeah native korean, almost people have phobia of english)